src/components/App.css

```diff
@@ -25,7 +29,7 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
-  padding: 20px 0px;
+  padding: 10px 0px;
 }
```

Comment:
are there any preferences on writing css?

I see mixtures of `em` and `px`, are there any guidelines I could have a look at?